evewspace/Map/models.py

```diff
@@ -597,6 +597,16 @@ def has_siblings(self):
             return False
         return parent_sys.childsystems.count() > 1
 
+    def distance_from_root(self):
```

Comment:
Distance calculation is now only used on signature edits if the MAP_ADVANCED_LOGGING setting is active (it's a new setting in General Settings).